share/spice/caniuse_lists/footer.handlebars

```diff
@@ -0,0 +1,36 @@
+<span class='align'>
```

Comment:
@moollaza I didn't understand this. I haven't tried to use the Grid classes in the footer..